src/Tasks/Common/Resources/xlf/Strings.de.xlf

```diff
@@ -1,354 +1,354 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
-  <file datatype="xml" source-language="en" target-language="de" original="../Strings.resx">
+  <file datatype="xml" source-language="en" target-language="de-DE" original="../Strings.resx">
```

Comment:
@khoiph1 I noticed in several of these that target-language is changed to a more specific culture than we actually use it as in the product. I would prefer if these continued to match the file name.

Reply:
I'm going to revert this for this insertion. I'm 99% sure nothing in the product is using target-language, but I don't want to take a change other than translation at the end of the release. We can talk about it on the next handback.